app/views/users/_form.html.haml

```diff
@@ -0,0 +1,35 @@
+= form_for(@user) do |f|
+
+  .row
+    .col.pe-6
+      .row
+        .col.pe-2
+          = f.label :name
+        .col
+          .value= f.object.full_name
+      .row
+        .col.pe-2
+          = f.label :telephone
+        .col
+          .value= f.object.telephone
+
+      .row
+        .col.pe-2
+          = f.label :email
+        .col
+          .value= f.object.email
+    .col.pe-6
+      .row
+        .col.pe-2
+          .value= f.label :is_active, "Status"
+        .col
+          = f.check_box :is_active
```

Comment:
If we did that then maybe could remove `can_edit?` from controller which I don't really like anyway?